public_html/wp-content/mu-plugins/camptix-tweaks/camptix-tweaks.php

```diff
@@ -919,6 +924,50 @@ function modify_shortcode_contents( $shortcode_contents, $tix_action ) {
 	return $shortcode_contents;
 }
 
+/**
+ * Show error message if IP Address has been throttled by `Form_Spam_Prevention`.
+ */
+function show_throttle_notice() {
+	global $camptix;
+
+	$fsp = new Form_Spam_Prevention(
+		[
+			'prefix' => WC_CAMPTIX_FSP_PREFIX,
+		]
+	);
```

Comment:
`MIGHT` - Since there's only 1 array item, and it's relatively short, it feels like this would be more readable if it were all on a single line. It'd only be about `75` chars, which is far under the `115` limit we typically use for wrapping.

If there were multiple non-trivial array items, though, then it probably would be better to wrap regardless of length.

Same applies to the other function as well.